### ISSUE TITLE
Forum Category Page

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/pioneer#3153]
## Context
Forum UX facelift upd

## Design Preview

[Link to Figma> ](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-Design?node-id=9364%3A339383)

![](https://user-images.githubusercontent.com/93924525/171611749-395475b8-274a-4e5b-a9ff-3a2339a66df3.png)

## Scope
:one: **Popular threads**
![](https://user-images.githubusercontent.com/93924525/171612242-f27c7b99-d2e5-41b1-ae01-09f9976bb592.png)

* [ ] Popular threads - Top 10 threads sorted by number of thread replies
* [ ] Carousel and card are same as Latest threads
* [ ] Fetch / show number based on screen width

:two: **Subcategories**
![](https://user-images.githubusercontent.com/93924525/171612180-ef116e33-9ce3-4bd5-adec-5a4a08a921b6.png)

* [ ] On hover show sub-categories
* [ ] On lowest level on hover keep showing description
* [ ] Categories should wrap on smaller screens

out of scope: ⛔ 
* [ ] ~Display 4 sub-categories in a grid and button "show all"~

:three: **Threads**

![](https://user-images.githubusercontent.com/93924525/171612195-e519e6e3-cb8b-4828-b13f-e080ab0a03f1.png)

* [ ] Table 
* [ ] Total counter on top as pill
* [ ] Truncated thread name
* [ ] Replies counter
* [ ] Last replies
* [ ] Author

Basically no change from production just copy pasta :) 

:four: 

* [ ] Search bar
* [ ] Moderators avas

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201958687417145/1202464660764776) by [Unito](https://www.unito.io)

**Comment by Ken-tech-max** [comment]
Hi, Isonar, Can you explain the issues one by one?
I want to ensure the issues and jump to development.

**Comment by gyroflaw** [comment]
hey I am gyroflaw#5905

**Comment by traumschule** [comment]
this was inactive for a bit, unassigning for now because #3029 is in `mainnet` scope. #3562 was reverted, @thesan how should we go from here?

**Comment by dmtrjsg** [comment]
@traumschule we've decided to finish the forum in the team internally, so pls feel free to ignore all Forum tasks for communty dev for the time being.

**Comment by dmtrjsg** [comment]
was done in a diff PR but worth testing based on issue description

**Comment by polikosi** [comment]
1️⃣Latest threads
✅Popular threads - Top 10 threads sorted by number of thread replies
![image](https://user-images.githubusercontent.com/80429191/205066213-47b11ada-0092-42c6-a70a-cfd2e2584d70.png)
✅Carousel and card are same as Latest threads

✅Fetch / show number based on screen width
1920×1080
![image](https://user-images.githubusercontent.com/80429191/205068631-13a100b2-9aca-47dc-99df-523dcf2a58e7.png)

1366×768
![image](https://user-images.githubusercontent.com/80429191/205068528-0af96c1a-cf6b-4c29-8c27-fe90d47b0331.png)

1440×900
![image](https://user-images.githubusercontent.com/80429191/205068775-948e2942-f3c4-47d4-b251-27d5413668b7.png)

1280×800
![image](https://user-images.githubusercontent.com/80429191/205068899-81f2b6f0-2608-48c2-8f48-b2b3284ff37a.png)

2️⃣ Subcategories
On hover show sub-categories
 ![image](https://user-images.githubusercontent.com/80429191/205069413-d7de33bc-7edc-4b87-856e-a2b45fec93cf.png)

✅On lowest level on hover keep showing description

Categories should wrap on smaller screens
![image](https://user-images.githubusercontent.com/80429191/205069904-9649bc3e-61c9-4b8b-9b82-f3590216cd2b.png)

3️⃣ Threads
![image](https://user-images.githubusercontent.com/80429191/205071280-d336428a-1a88-4c0b-bf47-67248360747c.png)

✅Table
✅Total counter on top as pill
✅Truncated thread name
![image](https://user-images.githubusercontent.com/80429191/205070524-f5e3f49c-a352-4157-8aaf-9e11443b4039.png)

✅Replies counter

Last replies
⚠️Shown "Last acticity"
![image](https://user-images.githubusercontent.com/80429191/205070758-e5fbaa0c-070f-4e99-889e-428e62ffec01.png)

✅Author

4️⃣
✅Search bar
![image](https://user-images.githubusercontent.com/80429191/205071706-e800f82c-9b79-4aa1-b474-f83fff8dd234.png)

⚠️Moderators avas
not showed moderators
![image](https://user-images.githubusercontent.com/80429191/205074536-6bf54774-b853-43c8-aefb-3dd14307a9df.png)

**Comment by dmtrjsg** [comment]
Looks like all of the found problems already raised as separate issues. this one is good to go then 👍
